packages/core/contracts/financial-templates/common/financial-product-libraries/PreExpirationIdentifierTransformationFinancialProductLibrary.sol

```diff
@@ -24,7 +23,6 @@ contract PreExpirationIdentifierTransformationFinancialProductLibrary is Financi
      */
     function setFinancialProductTransformedIdentifier(address financialProduct, bytes32 transformedIdentifier)
```

Comment:
nit unrelated to this line, comment on line 19 should say "pre" not "post". Also should remove language about the Owner.